src/components/PopupContainer/PopupContainer.js

```diff
@@ -3,13 +3,13 @@ import { Transition } from '@headlessui/react'
 import useLockBodyScroll from './../../utils/useLockBodyScroll.js'
 
 /**
- * Component that renders the PopupContainer component.
+ * Component that renders a PopupContainer component.
  *
  * @component
  *
- * @param {object} children - Parameter containing information that is displayed within the transition.
- * @param {boolean} show - Parameter that is used to show or hide the transitions.
- * @param {boolean} close - Parameter that is used to close the PopupContainer component when, the user clicks on the Escape key or if the user clicks outside the PopupContainer.
+ * @param {object} children - Parameter containing props that is used within the popup.
```

Comment:
children - can contain child components
See: https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891

Reply:
I've changed the parameter description.